BossMod/Autorotation/StatelessRotation.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace BossMod.Autorotation;
+public abstract class StatelessRotation(WorldState ws, float effectiveRange)
```

Comment:
What does this name mean? All rotations are stateless...

Reply:
yeah it's a shit name, this is just a convenience class for a rotation that doesn't interact with RotationModuleManager at all, like no strategy overrides or anything. it's specifically for rotations that need to be usable in QuestBattles as well as regular boss modules since all we have access to in QuestBattle is the world state. see Nyelbert's Lament

Reply:
Can it be moved to questbattles then?

Reply:
moved